### ISSUE TITLE
TypeDoc 0.25.x support

### ISSUE THREAD

**Issue by Maximvdw** [krisztianb/typedoc-umlclass#33]
The peer dependency declaration:
```json
  "peerDependencies": {
        "typedoc": "0.23.x || 0.24.x"
   }
```
breaks compatibility with typedoc 0.25 due to dependency conflicts

**Comment by krisztianb** [comment]
Yes I know. :smile: That is intentional because major TypeDoc updates usually break plugins.

Have you tried if the plugin still works with version 0.25.x of TypeDoc? That peer dependency message you get is just a warning if I'm not mistaken.

I still have to find the time to look into the new TypeDoc version and whether it is compatible with this plugin or not. I'm especially concerned about this change: https://github.com/TypeStrong/typedoc/pull/1461

**Comment by Maximvdw** [comment]
hey, there are no issues in 0.25.x. To be fair, it would be more developer friendly to not enforce the peer dependencies to break compatibility before a version is actually released.

**Comment by krisztianb** [comment]
Thank you for the update.

I think it is not a good idea to include version numbers in a peer dependency that might not work with the plugin. The good solution here would be to be faster in releasing new versions of the plugin. However I'm personally not using TypeDoc anymore and have other more important stuff to do. So while this has no priority for me I still try to keep the package up to date.

**Comment by krisztianb** [comment]
I guess you are using the plugin with the `location: remote` setting? Unfortunately all other location settings are broken and no longer work with TypeDoc 0.25.x. Will have to take a deeper dive into this.

**Comment by krisztianb** [comment]
New version **0.8.0** released which supports TypeDoc 0.25.x, but no longer supports TypeDoc 0.23.x. Feel free to report any bugs you find. Happy testing. 🎉

**Comment by Maximvdw** [comment]
I was indeed using location: remote, but for another project it was indeed not working on 'local'. Sorry for not seeing that sooner.

The new version seems to work fine. Thank you!
